### ISSUE TITLE
On Pruned bitcoind support

### ISSUE THREAD

**Issue by nickodev** [Samourai-Wallet/samourai-dojo#87]
I had my dojo connected to a _pruned_ bitcoind on a raspberry pi the other day and it seems to be working just fine. The node container successfully imported all the blocks' headers into mysql db container and I could pair my mobile Samourai-Wallet and Whirlpool-gui with it. Everything works as if my bitcoind was an archiving node. 

**My question is: If users don't want to import their existing seeds/keys; is it correct to say Dojo supports pruned bitcoind out of the box?**

I believe supporting pruned bitcoind will bring more users to the Samourai ecosystem.

**Comment by LaurentMT** [comment]
Hi nickodev.

As far as I can tell, current versions of Dojo require a non pruned full node. One of the requirements for Dojo is a full node running with the txindex activated and if I'm correct it isn't compatible with the pruned mode. Thus, I wouldn't be surprised if you start to have issues at some point.

To be honest, I don't think that supporting a pruned mode is the direction currently followed by Dojo. Indeed, a pruned node creates challenges for several existing features (import/rescan of xpubs/addresses with past activty, sweep of privkeys from others wallets, etc).

In conclusion, I wouldn't say that support of a pruned mode is definitely excluded but it's clearly not in the "high priority tasks" list for now.

**Comment by nickodev** [comment]
Hi LaurentMT,

Actually I reviewed the code and logs and the current version of the Dojo develop branch supports the pruned node. The only trade-off is users can not import their seeds/keys into it (obviously because of the missing txindex) but if someone like me wants to just fresh start with a new seed, it is absolutely compatible. I am currently running the develop version of the Dojo along with a pruned(5GB) bitcoin-core on a raspberry-pi and managed to pair my wallet and my whirlpool-gui with it and they work without any problem. 

I am going to close this issue for now. will test it further and maybe someday we will promote it in the documentations.
